index.html

```diff
@@ -68,7 +68,7 @@ <h1 class="light copy">Track GeoSpatial Data Edits</h1>
 
             </div>
             <div class="col-md-offset-1 col-md-4">
-              <a href="http://download.locationtech.org/geogig/geogig-1.0-RC4.zip" class="btn download"> <span class="zipfile"></span> <div class="download-ziptxt">DOWNLOAD <br/>1.0-RC4 Release .zip</div></a>
+              <a href="https://www.locationtech.org/projects/technology.geogig/downloads/geogig-1.0.zip" class="btn download"> <span class="zipfile"></span> <div class="download-ziptxt">DOWNLOAD <br/>1.0 Release .zip</div></a>
               <a href="https://github.com/locationtech/geogig/releases/download/v1.0-RC4/gs-geogig-2.9.zip" class="btn download"> <span class="zipfile"></span> <div class="download-ziptxt">DOWNLOAD<br/>GeoServer 2.9 Plugin<br/></div></a>
```

Comment:
These are available from the github download page here: 

https://github.com/locationtech/geogig/releases/tag/v1.0